modules/govuk_postgresql/manifests/db.pp

```diff
@@ -158,5 +159,12 @@
       collectd::plugin::postgresql_db{$db_name:}
       govuk_postgresql::monitoring::db{$db_name:}
     }
+  } else {
+    if (!empty($extensions)) {
+      postgresql::server::extension { $extensions:
```

Comment:
@deanwilson if `$extensions` is `[]` then this will not do anything so the `if (!empty..` guard doesn't need to exist, amirite?

Reply:
Hmm, yeah, I would assume so... I included it as the one for non RDS does it for some reason.

https://github.com/alphagov/govuk-puppet/pull/7289/files#diff-127ce7effed9df8b23c79cefcf3195b4R125